insights_client/__init__.py

```diff
@@ -30,13 +31,16 @@
 try:
     insights_uid = pwd.getpwnam("insights").pw_uid
     insights_gid = pwd.getpwnam("insights").pw_gid
+    insights_grpid = grp.getgrpname("insights").gr_gid
+    insights_grpusers = grp.getgrpname("insights").gr_mem
```

Comment:
Is this used anywhere?

Reply:
insights_grpusers is not used anywhere so I can remove that.